packages/create-start/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@tanstack/create-start",
+  "version": "1.81.5",
+  "description": "Modern and scalable routing for React applications",
+  "author": "Tim O'Connell",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/TanStack/router.git",
+    "directory": "packages/create-router"
+  },
+  "homepage": "https://tanstack.com/router",
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/tannerlinsley"
+  },
+  "bin": {
+    "create-router": "index.js"
+  },
+  "scripts": {
+    "dev": "unbuild --stub --watch",
+    "clean": "rimraf ./dist && rimraf ./coverage",
+    "test:eslint": "eslint ./src",
+    "generate-templates": "node ./dist/generate-templates/index.mjs",
+    "build": "unbuild",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
+    "test:coverage": "vitest run --coverage"
+  },
+  "type": "module",
+  "files": [
+    "index.js",
+    "templates",
+    "dist"
+  ],
+  "engines": {
+    "node": "^18.0.0 || >=20.0.0"
+  },
+  "exports": {
+    ".": {
+      "import": "./src/index.ts"
+    }
+  },
+  "devDependencies": {
+    "@commander-js/extra-typings": "^12.1.0",
+    "@inquirer/prompts": "^5.5.0",
+    "@inquirer/type": "^3.0.1",
+    "@playwright/test": "^1.48.2",
+    "@types/cross-spawn": "^6.0.6",
+    "@types/validate-npm-package-name": "^4.0.2",
+    "cross-spawn": "^7.0.5",
+    "fast-glob": "^3.3.2",
+    "get-port-please": "^3.1.2",
+    "gradient-string": "^3.0.0",
+    "picocolors": "^1.1.1",
+    "tempy": "^3.1.0",
+    "terminate": "^2.8.0",
+    "tiny-invariant": "^1.3.3",
+    "unbuild": "^2.0.0",
+    "validate-npm-package-name": "^5.0.1",
+    "wait-port": "^1.1.0",
+    "yocto-spinner": "^0.1.1",
+    "zod": "^3.23.8",
+    "zod-opts": "^0.1.8"
+  },
+  "peerDependencies": {
+    "@tanstack/react-router": "workspace:^",
+    "@tanstack/router-devtools": "workspace:^",
+    "@tanstack/start": "workspace:^",
+    "@types/react": "^18.3.12",
+    "@types/react-dom": "^18.3.1",
+    "react": "^18.3.1",
```

Comment:
how would we handle an option that lets the user choose between react18 and react19?

Reply:
That's only used to get the semver. I think we can cross that bridge when we come to it - a work around or a new process for this case